modules/core/src-native/weaver/internals/Timestamp.scala

```diff
@@ -0,0 +1,35 @@
+package weaver.internals
+
+import scala.scalanative.posix
+import scala.scalanative.unsafe._
+
+import posix.time
+import posix.timeOps._
+
+private[weaver] object Timestamp {
+
+  def format(epochSecond: Long): String = Zone { implicit zone =>
+    val out     = alloc[time.tm]()
+    val timePtr = alloc[time.time_t]()
+    !timePtr = epochSecond
+    val gmTime: Ptr[time.tm] = time.localtime_r(timePtr, out)
+    val hour                 = gmTime.tm_hour
+    val minutes              = gmTime.tm_min
+    val seconds              = gmTime.tm_sec
+    s"$hour:$minutes:$seconds"
+  }
+
+  def localTime(hours: Int, minutes: Int, seconds: Int): Long = Zone {
+    implicit zone =>
+      val out     = alloc[time.tm]()
```

Comment:
same as above - the return is `Long`, by value, so the memory can be allocated by `stackalloc` - this would make it much, much faster as well

Reply:
Addressed [aa7296e](https://github.com/disneystreaming/weaver-test/pull/568/commits/aa7296e2be5a9ba2cd6ccad7cda27c5b76c76b81)